contrib/aws/aws-sdk-go-v2/aws/aws.go

```diff
@@ -136,10 +139,19 @@ func (mw *traceMiddleware) deserializeTraceMiddleware(stack *middleware.Stack) e
 	}), middleware.Before)
 }
 
-func serviceName(cfg *config, serviceID string) string {
+func getSpanName(awsService, awsOperation string) string {
+	getSpanNameV0 := func(awsService string) string { return awsService + ".request" }
```

Comment:
Seems fine to just call this `fn` since it's only used once immediately after it's defined. Or honestly just inlining this would probably be fine:
```
return awsnamingschema.NewAWSOutboundOp(service, operation, func(s string) string {
    return s + ".request"
}).GetName()
```